test/unit/StakeDaoCurveStrategyOperator.unit.ts

```diff
@@ -0,0 +1,617 @@
+import { expect } from "chai";
+import { createFixtureLoader } from "ethereum-waffle";
+import { BigNumber, utils, Wallet } from "ethers";
+import { ethers } from "hardhat";
+import { cleanResult, getStakeDaoDepositETHOrder, getStakeDaoDepositOrder, getStakeDaoWithdraw128Order, getStakeDaoWithdrawETHOrder, OrderStruct } from "../../scripts/utils";
+import { appendDecimals, BIG_NUMBER_ZERO, getExpectedFees, UINT256_MAX } from "../helpers";
+import { factoryAndOperatorsForkingBSCFixture, FactoryAndOperatorsForkingBSCFixture, factoryAndOperatorsForkingETHFixture, FactoryAndOperatorsForkingETHFixture, USDC } from "../shared/fixtures";
+import { describeOnBscFork, describeOnEthFork, provider } from "../shared/provider";
+import { LoadFixtureFunction } from "../types";
+
+let loadFixture: LoadFixtureFunction;
+
+describeOnBscFork("StakeDaoCurveStrategyOperator BSC fork", () => {
+    let context: FactoryAndOperatorsForkingBSCFixture;
+    const ETH = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
```

Comment:
Are we sure ETH has the same address on BSC ?
I though it was [this one](https://bscscan.com/token/0x2170ed0880ac9a755fd29b2688956bd959f933f8) 
I might miss something !

Reply:
The variable `ETH` dosen't refer to the Binance-Peg Ethereum Token, but to the BNB address.
Whatever the chain, ETH always refers to the chain native crypto.

Reply:
Oh right 
Thank you !